src/xrpld/app/tx/detail/VaultWithdraw.cpp

```diff
@@ -178,7 +178,7 @@ VaultWithdraw::doApply()
         return tefINTERNAL;  // LCOV_EXCL_LINE
 
     auto const mptIssuanceID = (*vault)[sfShareMPTID];
-    auto const sleIssuance = view().read(keylet::mptIssuance(mptIssuanceID));
+    auto const sleIssuance = view().read(keylet::mptIssuance(*mptIssuanceID));
```

Comment:
Same as in `VaultClawback`, `mptIssuanceID` can be declared with the `*`.

Reply:
thanks, implemented in https://github.com/XRPLF/rippled/pull/5652/commits/6893015759ad8b7041611551c453b6efd9ac9051 (with extra parents for readability)